src/@sdk/react/components/CredentialsProvider/CredentialsProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import { useAuth, useSignIn } from "../..";
+import { IProps } from "./types";
+
+export function CredentialsProvider({
+  children,
+}: IProps): React.ReactElement<IProps> {
+  const { authenticated } = useAuth();
+  const [signIn] = useSignIn();
+
+  const autoSignIn = async () => {
+    const credentials = await navigator.credentials.get({
+      mediation: "optional",
+      password: true,
+    });
+
+    if (credentials) {
+      await signIn({
+        email: credentials.id,
+        password: credentials.password,
+      });
+    }
+  };
+
+  React.useEffect(() => {
+    if (!authenticated && window.PasswordCredential) {
```

Comment:
navigator check needed here as well, do we need to check for `window.PasswordCredential` since we don't use it in autoSignIn func?

Reply:
There is no need to check the navigator object. According to the docs, we should check the `window.PasswordCredential` object to detect CM support.